errors/errors.go

```diff
@@ -0,0 +1,108 @@
+package errors
```

Comment:
would have preferred to keep this file in the root directory of the project.

Reply:
moved in root.